modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/DataLayoutBuilderInstanceProvider.es.js

```diff
@@ -87,6 +88,9 @@ export default ({children, dataLayoutBuilder}) => {
 
 		const deleteFromObjectAction = {
 			action: (event) => deleteDefinitionFieldModal(event),
+			disabled:
+				focusedField?.nativeField ||
```

Comment:
Is it possible for the field to have the nativeField property directly?

Reply:
It may not exist a focusedField, but if the focusedField default is an empty object, can work fine.

What you think?

Reply:
I don't think we need this condition because the `nativeField` property exists inside `customProperties`